static/app/views/profiling/profileGroupProvider.tsx

```diff
@@ -103,4 +99,4 @@ function ProfileGroupProvider(props: FlamegraphViewProps): React.ReactElement {
   );
 }
 
-export default ProfileGroupProvider;
+export default React.memo(ProfileGroupProvider);
```

Comment:
can we get rid of the use of `import React from 'react'` since we're here?

Reply:
I'll drop the whole memo. This is the artifact of me trying to figure out why the page was reloading 😅